system/jlib/jset.hpp

```diff
@@ -126,6 +126,24 @@ inline unsigned getMostSignificantBit(unsigned value)
 #endif
 }
 
+#if defined (_WIN32)
+// These are standard in glibc
+inline int ffsll(__uint64 i)
+{
+    __uint64 x = i & -i;
```

Comment:
return i ? countTrailingUnsetBits(i) + 1 : 0;

Reply:
Hmmm I missed that. I will change.